pandapower/estimation/algorithm/base.py

```diff
@@ -83,6 +86,7 @@ def estimate(self, eppci: ExtendedPPCI, **kwargs):
 
         current_error, cur_it = 100., 0
         # invert covariance matrix
+        eppci.r_cov[eppci.r_cov<(10**(-6))] = 10**(-6)
```

Comment:
This value of 10-6 should ideally be parameterizable

Reply:
It is an option I am considering, but I am not yet sure if to put it as an optional parameter for the user or not (I see some pros and cons). In any case, this will be definitely improved in the next round of updates, for the moment it was urgent to avoid having 0 or close to 0 values in the covariance matrix.

Reply:
This is central to the feasibility of the solution and to evaluate how good the estimate is and where are problems in the network, so one could change it and then simulate multiple usecases